modules/table/src/testFixtures/java/org/apache/ignite/internal/table/TableTestUtils.java

```diff
@@ -105,6 +105,16 @@ public static void dropTable(CatalogManager catalogManager, String schemaName, S
         );
     }
 
+    /**
+     * Drops table from {@link #createSimpleTable(CatalogManager, String)} in the catalog.
+     *
+     * @param catalogManager Catalog manager.
+     * @param tableName Table name.
+     */
+    public static void dropSimpleTable(CatalogManager catalogManager, String tableName) {
```

Comment:
Why is this method needed? Why is it called `dropSimpleTable` and not `dropTable`? What's so "simple" about it?

Reply:
This method was created for convenience in order to write less test code of the same type. 
It is convenient if the table was created using `TableTestUtils#createSimpleTable`, as the documentation says.

Reply:
why not call it `dropTable` then?

Reply:
To make it easier to understand by names, we created a simple table, deleted the simple table, it’s clearer.